Dockerfile

```diff
@@ -1,5 +1,17 @@
+FROM golang:alpine as builder
+
+WORKDIR /go/src/github.com/blendle/pg2kafka
+ADD . ./
+
+RUN apk --update --no-cache add git alpine-sdk bash
+RUN wget -qO- https://github.com/edenhill/librdkafka/archive/v0.11.4-RC1.tar.gz | tar xz
+RUN cd librdkafka-* && ./configure && make && make install
+RUN go get github.com/golang/dep/cmd/dep && dep ensure -vendor-only
```

Comment:
You could add the Gopkg files separately to improve build with cached layers: https://github.com/blendle/bundle-processors/pull/143/files#diff-27432990f383ad03bfcec79176df72a7R5

PS you could also use the base image `eu.gcr.io/bnl-blendle/docker-golang-librdkafka:1.10-alpine as builder` which has all the librdkafka stuf already present

Reply:
I could get that to work, `dep` also looks at your source right?

Can't use blendle images, this is an opensource tool